scripts/build_dists_zip.sh

```diff
@@ -199,53 +203,77 @@ echo "Hash file $hashFile"
 echo "Archive file $archiveFile"
 
 distDir=$(mktemp -d)
-if [ -d $distDir ]; then
-  echo "Temporary directory $distDir exists"
+if [ -d ${distDir} ]; then
+  echo "Temporary directory ${distDir} exists"
+  mkdir -p ${distDir}/runtime
+  mkdir -p ${distDir}/config
 else
-  echo "Temporary directory $distDir does not exist"
+  echo "Temporary directory ${distDir} does not exist"
   exit 6
 fi
 
-mkdir $distDir/dist
-
-COPY_FILES=(
-  "target/release/tari_base_node"
-  "target/release/tari_console_wallet"
-  "target/release/tari_merge_mining_proxy"
-  "target/release/tari_mining_node"
-  "common/config/presets/tari_sample.toml"
-  "common/config/tari_config_example.toml"
-  "common/logging/log4rs_sample_base_node.yml"
-  "applications/tari_base_node/README.md"
-  applications/tari_base_node/$osname/*
-  "scripts/install_tor.sh"
-)
-
-for COPY_FILE in "${COPY_FILES[@]}"; do
-  cp -vr "$COPY_FILE" "$distDir/dist/"
-done
-
-pushd $distDir/dist
+# One click miner
+cp -f -P "${PWD}/applications/tari_base_node/osx/start_all" "${distDir}/start_all"
```

Comment:
Will the Ubuntu Archive be including the OSX files? Are the OSX files and Ubuntu the same? I have not checked.

Reply:
There is a `if [ "$osname" == "osx" ]; then` a bit further down, missed these for some reason, will fix. Thanks.